src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_COST = 40;
+  const BIG_DISCOUNT = 50;
+  const SMALL_DISCOUNT = 20;
+
+  const PERIOD_WITHOUT_DISCOUNT = 2;
+  const PERIOD_WITH_SMALL_DISCOUNT = 7;
+
+  let price = days * RENT_COST;
+
+  if (days <= PERIOD_WITHOUT_DISCOUNT) {
+    return price;
+  }
+
+  if (days < PERIOD_WITH_SMALL_DISCOUNT) {
+    return (price -= SMALL_DISCOUNT);
+  }
+
+  return (price -= BIG_DISCOUNT);
```

Comment:
The function name 'calculateRentalCost' is descriptive and clear, which is good. However, the constants' names could be more descriptive to reflect the context of the rental period they represent. For example, 'PERIOD_WITH_SMALL_DISCOUNT' could be misleading as it suggests the period itself has a discount, rather than being the threshold for a discount. Consider renaming it to something like 'LONG_TERM_RENTAL_THRESHOLD' to better convey its purpose.